### DISCUSSION TITLE
user-resizable containers

### DISCUSSION THREAD

**Discussion by danchitnis** [chakra-ui/chakra-ui#5113] [Q&A] [answered]
Hello, Is there a way to make user-resizable containers as described in the article below within Chakra? Thanks for your help.
https://dev.to/dailydevtips1/making-divs-user-resizable-with-css-3ga

**Comment by primos63** [comment, marked as answer]
```jsx
 <Box resize='both' width='100px' height='100px' overflow='auto' bg='pink.400' />
```

**Reply by danchitnis** [reply]
Thanks, it works 👍

**Reply by ioswmtdeveloper** [reply]
Thanks.

**Reply by josephp27** [reply]
is there a way to make draggable handle the entire side of the box not the bottom right. like in the case of a resizable sidebar where you can drag anywhere on the right side

**Reply by danchitnis** [reply]
@josephp27 It would be nice if someone can make that example (i.e. the resizable sidebar as you explained)
